doc/release-notes-1.2.0.rst

```diff
@@ -0,0 +1,550 @@
+* The :ref:`gps_with_supl_support_sample` sample stops working if :ref:`supl_client` support is enabled, but the SUPL host name cannot be resolved.
+  As a workaround, insert a delay (``k_sleep()``) of a few seconds after the ``printf`` on line 294 in :file:`main.c`.
+* Deprecation warning: The nrf_inbuilt_key API in the :ref:`nrfxlib:bsdlib` will be removed in a future release.
+  A replacement library that wraps the AT commands for ``AT%CMNG`` will be available in the |NCS|.
+* The :ref:`asset_tracker` sample might show up to 2.5 mA current consumption in idle mode with ``CONFIG_POWER_OPTIMIZATION_ENABLE=y``.
+* The SEGGER Control Block cannot be found by automatic search by the RTT Viewer/Logger.
+  As a workaround, set the RTT Control Block address to 0 and it will try to search from address 0 and upwards.
+  If this does not work, look in the ``builddir/zephyr/zephyr.map`` file to find the address of the ``_SEGGER_RTT`` symbol in the map file and use that as input to the viewer/logger.
+* nRF91 fails to receive large packets (over 4000 bytes).
+* nrf_connect fails if called immediately after initialization of the device.
+  A delay of 1000 ms is required for this to work as intended.
+
+
+Subsystems
+==========
+
+Bluetooth Low Energy
+--------------------
+
+* The :ref:`peripheral_hids_keyboard` sample cannot be used with the :ref:`nrfxlib:ble_controller` because the NFC subsystem does not work with the controller library.
+  The library uses the MPSL Clock driver, which does not provide an API for asynchronous clock operation.
+  NFC requires this API to work correctly.
+* When the :ref:`peripheral_hids_mouse` sample is used with the Zephyr Bluetooth LE Controller, directed advertising does not time out and the regular advertising cannot be started.
+* The :ref:`bluetooth_central_hids` sample cannot connect to a peripheral that uses directed advertising.
+* Bluetooth Low Energy peripheral samples are unstable in some conditions (when pairing and bonding are performed and then disconnections/re-connections happen).
+* When running the :ref:`bluetooth_central_dfu_smp` sample, the :option:`CONFIG_BT_SMP` configuration must be aligned between this sample and the Zephyr counterpart (:ref:`zephyr:smp_svr_sample`).
+  However, security is not enabled by default in the Zephyr sample.
+* The central samples (:ref:`central_uart`, :ref:`bluetooth_central_hids`) do not support any pairing methods with MITM protection.
+* On some operating systems, the nrf_desktop application is unable to reconnect to a host.
+* central_uart: A too long 212-byte string cannot be handled when entered to the console to send to peripheral_uart.
+* On nRF51 devices, Bluetooth LE samples that use GPIO might crash when buttons are pressed frequently.
+  In such case, the GPIO ISR introduces latency that violates real-time requirements of the Radio ISR.
+  nRF51 is more sensitive to this issue than nRF52 (faster core).
+
```

Comment:
Not sure about the best placement for this:

> BLE cannot be used in a non-secure app, i.e. the nrf5340_dk_nrf5340_cpuappns board. Use the nrf5340_dk_nrf5340_cpuapp board instead.

cc @bjorn-tore-taraldsen @rugeGerritsen

Reply:
@grochu Ref. our discussion aboyt Sec/ non-se. This is the conclusion from Øyvind. Any comment

Reply:
We might want to add a note or update somehow the nRF53 User Guide.

Reply:
Too late for doc updates now - do we expect this to be a persistent problem, or will we fix it?

Added to the BLE section now.